src/signature/generalized_xmss.rs

```diff
@@ -666,17 +669,15 @@ mod tests {
         assert_eq!(rho1, rho2);
     }
 
-    /*#[test]
-    pub fn test_large_base_sha() {
+    #[test]
+    pub fn test_large_base_poseidon() {
         // Note: do not use these parameters, they are just for testing
-        type PRF = ShaPRF<24, 8>;
-        type TH = ShaTweak192192;
-
-        // use chunk size 8
-        type MH = ShaMessageHash<24, 8, 32, 8>;
-        const TARGET_SUM: usize = 1 << 12;
+        type PRF = ShakePRFtoF<4, 4>;
+        type TH = PoseidonTweakHash<4, 4, 2, 8, 8>;
+        type MH = PoseidonMessageHash<4, 4, 2, 8, 256, 2, 9>;
+        const TARGET_SUM: usize = 8 * (256 - 1) / 2;
         type IE = TargetSumEncoding<MH, TARGET_SUM>;
-        const LOG_LIFETIME: usize = 10;
+        const LOG_LIFETIME: usize = 6;
```

Comment:
Why can't we use this?

```suggestion
        const LOG_LIFETIME: usize = 10;
```